frontend/src/components/classify-failures.js

```diff
@@ -248,35 +199,25 @@ export class ClassifyFailuresTable extends React.Component {
       rows,
       selectedResults,
       includeSkipped,
-      isExceptionOpen,
-      exceptionSelections,
-      exceptions,
     } = this.state;
+    const {
+      run_id
+    } = this.props
     const pagination = {
       pageSize: this.state.pageSize,
       page: this.state.page,
       totalItems: this.state.totalItems
     }
-    // filters for the exception
-    const exceptionFilters = [
-      <React.Fragment key="value">
-        <Select
-          aria-label="exception-filter"
-          placeholderText="Filter by exception"
-          variant={SelectVariant.checkbox}
-          isOpen={isExceptionOpen}
-          selections={exceptionSelections}
-          maxHeight={"1140%"}
-          isDisabled={exceptions.length < 2}
-          onToggle={this.onExceptionToggle}
-          onSelect={this.onExceptionSelect}
-          onClear={this.onExceptionClear}
-        >
-          {exceptions.map((option, index) => (
-            <SelectOption key={index} value={option._id} description={option.count + ' results'}/>
-          ))}
-        </Select>
-      </React.Fragment>
+    // filters for the metadata
+    const resultFilters = [
+      <MetaFilter
+        key="metafilter"
+        // user_properties fields shouldn't be injected here
+        fieldOptions={['metadata.user_properties.assignee', 'metadata.user_properties.component', ...STRING_RESULT_FIELDS,]}
```

Comment:
There are some `fieldOptions` like `run_id` and `project_id` which are included in `STRING_RESULT_FIELDS` that aren't really relevant here. 

I think we should just leave with a few fields for now. Also, I noticed `metadata.exception_name` isn't included in the list.

Reply:
One piece that I _might_ leave as a TODO here is more abstract handling of adding fields to this list.

Similar to the values select, we can try to resolve a set of fields from the result items metadata - to include things like user_properties without having to enumerate beforehand.